packages/app/src/AppNavbar.tsx

```diff
@@ -91,8 +91,29 @@ export function AppNavbar({ closeNavbar }: AppNavbarProps): JSX.Element {
     }
   }
 
+  function navigateResourceType(resourceType: string | undefined): void {
+    if (resourceType) {
+      navigate(`/${resourceType}`);
+    }
+  }
+
   return (
     <Navbar width={{ sm: 250 }} p="xs">
+      <Navbar.Section>
+        <CodeInput
+          name="resourceType"
+          placeholder="Navigate by Resource Type"
+          property={{
+            binding: {
+              valueSet: 'http://hl7.org/fhir/ValueSet/resource-types',
```

Comment:
Unfortunately, all FHIR identifier URLs use "http" rather than "https", so this is a relatively common false positive.

Do you have access to the "Review" button in Sonar?  You should be able to dismiss with a comment such as "Only an identifier, not a request URL".

<img width="865" alt="image" src="https://user-images.githubusercontent.com/749094/232824528-e8861247-177f-47cc-adcb-37b1f36c04ee.png">